core-bundle/src/EventListener/DoctrineSchemaListener.php

```diff
@@ -30,5 +32,19 @@ public function __construct(private DcaSchemaProvider $provider)
     public function postGenerateSchema(GenerateSchemaEventArgs $event): void
     {
         $this->provider->appendToSchema($event->getSchema());
+
+        foreach (['contao_prio_high', 'contao_prio_medium', 'contao_prio_low'] as $transportName) {
```

Comment:
Hardcoded here. I think if you define additional transports yourself, you should also add them to the schema yourself.